src/target/nrf51.c

```diff
@@ -103,87 +105,40 @@ static void nrf51_add_flash(target *t,
 
 bool nrf51_probe(target *t)
```

Comment:
I think it should be mapping the FICR/UICR (as it did before) - it's really useful to be able to use `print` in gdb to examine these registers.

Reply:
FICR is some non-changeable memory area. No need to map it. You can always use examine  "x ..." to print. I re-added UICR, as it might get flashed.